encoding/layout/infix.rs

```diff
@@ -72,6 +73,7 @@ impl Infix {
         PropertyLabel => [0];
         PropertyValueType => [1];
         PropertyOrdering => [2];
+        PropertyOverride => [3];
 
        // Reserve: range 50 - 99 to store annotations with a value type - see InfixID::<CONSTANTS>
         PropertyAnnotationAbstract => [50];
```

Comment:
Question here. If we're reserving ranges, should we reserve from 50 or, say `0x40` ?
I've mostly seen these written as hex literals and printed as hex so you can tell the type from the hex code.

Reply:
I don't that idea but I just find reasoning over decimal numbers (reserved range 50 - 100) much easier than working in hex (extra mental step to think what range that is)